app/components/containerLogs/containerlogs.html

```diff
@@ -31,7 +31,7 @@
       </rd-widget-taskbar>
       <rd-widget-body classes="no-padding">
         <div class="panel-body">
-          <pre id="stdoutLog" class="pre-scrollable pre-x-scrollable">{{stdout}}</pre>
+          <pre id="stdoutLog" class="pre-scrollable pre-x-scrollable" ng-bind-html="stdout"></pre>
```

Comment:
Why do we need the `ng-bind-html` directive here?

Reply:
stdout contains html :
https://docs.angularjs.org/api/ng/directive/ngBindHtml

> You may also bypass sanitization for values you know are safe. To do so, bind to an explicitly trusted value via $sce.trustAsHtml. 
ngBindHtml should be used with $sce.trustAsHtml

Reply:
The goal of this PR is to introduce color support, I don't see why we should add HTML support in the logs.

This could lead to malicious code being introduced when displaying the logs of a container outputting HTML (and Javascript).

Reply:
yep sorry i changed it